src/services/cube-handler.ts

```diff
@@ -2059,6 +2067,48 @@ export const createFilesForDownload = async (quack: Database, datasetId: string,
   logger.debug('Async processes completed.');
 };
 
+export const createMaterialisedView = async (revisionId: string) => {
+  const connection = await getCubeDB().connect();
+  await connection.query(pgformat(`SET search_path TO %I;`, revisionId));
+  logger.info(`Creating default views...`);
+  const viewCreation = performance.now();
+  // Build the default views
+  try {
+    for (const locale of SUPPORTED_LOCALES) {
+      const lang = locale.toLowerCase().split('-')[0];
+
+      const defaultViewSQL = pgformat(
+        'CREATE MATERIALIZED VIEW %I AS SELECT * FROM %I;',
+        `default_mat_view_${lang}`,
+        `default_view_${lang}`
+      );
+      logger.debug(defaultViewSQL);
+      await connection.query(defaultViewSQL);
+
+      const rawViewSQL = pgformat(
+        'CREATE MATERIALIZED VIEW %I AS SELECT * FROM %I;',
+        `raw_mat_view_${lang}`,
+        `raw_view_${lang}`
+      );
+      logger.debug(rawViewSQL);
+      await connection.query(rawViewSQL);
+    }
```

Comment:
Future PR - might be worth considering building these in parallel with something like

```ts
await Promise.all(SUPPORTED_LOCALES.map(locale => {
 ...
});

```
as it might actually be quicker if the DB can handle it.